src/components/CodeAssignmentModal/index.jsx

```diff
@@ -97,6 +99,19 @@ export class BaseCodeAssignmentModal extends React.Component {
     this.setState({ mode });
   }
 
+  getCleanedUsersDetails(emails, usersDetailResponse) {
```

Comment:
can we rename from `getCleanedUsersDetails ` to `getCleanedUserDetails`
and 
`usersDetailResponse` to `userDetailsResponse`